app/code/Magento/Sales/Setup/UpgradeData.php

```diff
@@ -66,26 +49,17 @@ class UpgradeData implements UpgradeDataInterface
      * @param SalesSetupFactory $salesSetupFactory
      * @param Config $eavConfig
      * @param AggregatedFieldDataConverter $aggregatedFieldConverter
-     * @param AddressCollectionFactory $addressCollFactory
-     * @param OrderFactory $orderFactory
-     * @param QuoteFactory $quoteFactory
      * @param State $state
      */
     public function __construct(
         SalesSetupFactory $salesSetupFactory,
         Config $eavConfig,
         AggregatedFieldDataConverter $aggregatedFieldConverter,
-        AddressCollectionFactory $addressCollFactory,
```

Comment:
Removing constructor dependencies is not backward compatible change, please revert it